packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/NamedTypeSymbolProvider.cs

```diff
@@ -88,12 +154,20 @@ private static CSharpType GetCSharpType(ITypeSymbol typeSymbol)
                         : null);
             }
 
-            var type = System.Type.GetType(fullyQualifiedName);
+            Type? type = System.Type.GetType(fullyQualifiedName);
             if (type is null)
             {
                 throw new InvalidOperationException($"Unable to convert ITypeSymbol: {fullyQualifiedName} to a CSharpType");
             }
```

Comment:
is this temporary?
Because a symbol could be a partial class of an existing generated model/enum/client, which definitely returns null when we call `System.Type.GetType(fullQualifiedName)`